modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -98,4 +98,12 @@ With this update, this issue is fixed. This update resolves the issue by integra
 
 * There is currently a known issue that assigns a lower `SecurityContextConstraints` (SCC) to Redis service account in {gitops-shortname} v1.16 after upgrading from {gitops-shortname} v1.15 to v1.16. The {gitops-shortname} Operator does not update the `securityContext` of the `redis-ha-server` StatefulSet, which causes the container's user to be statically set instead of being randomly assigned as required by the `restricted-v2` SCC. The `redis-ha-server` StatefulSet pods retain the old configurations and are not updated with the new settings. As a result, the new configuration of the StatefulSet is not applied correctly.
 +
-Workaround: Manually delete the `redis-ha-server` StatefulSet to trigger the re-creation of the pods with the updated settings. link:https://issues.redhat.com/browse/GITOPS-6670[GITOPS-6670]
\ No newline at end of file
+Workaround: Manually delete the `redis-ha-server` StatefulSet to trigger the re-creation of the pods with the updated settings. link:https://issues.redhat.com/browse/GITOPS-6670[GITOPS-6670]
```

Comment:
PTAL at the repo guidelines topic for API object references: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

In summary, you can use the term "stateful set" as a generic reference, which is the preferred usage. Or you can use `StatefulSet` followed  by a noun, such as resource or object, to refer to the specific API object. As a rule of thumb, any use of camel case needs to be wrapped in backticks and followed by a noun or broken up to not be camel case for the generic reference.

Reply:
Thanks for the detailed explanation regarding the usage of backticks required in this instance. I have added backticks as you suggested.

Reply:
Thanks. Please make sure that if you use the specific API reference, the instance in backticks must be followed by a noun. For example, "`StatefulSet` resource" or "`StatefulSet` object" (IDK what the proper term is to follow).

Lastly, I forgot to mention to please squash your commits before asking for your merge review. Thank you!